docs/semgrep-ci/running-semgrep-ci-with-semgrep-app.md

```diff
@@ -328,11 +329,107 @@ SEMGREP_PR_ID="44"
 
 ### Receiving PR or MR comments
 
+To receive PR or MR comments in your repository, follow the steps to enable hyperlinks. Verify that comments are sent by adding rules to your Rule Board's **Comment** column that can match code to generate a finding.
+
 :::info
-* Semgrep Cloud Platform does not support PR comments within BitBucket repositories.
+Only rules that are in the **Comment** column of your [Rule board](https://semgrep.dev/orgs/-/board) create the PR comments.
 :::
 
-To receive PR or MR comments in your repository, follow the steps to enable hyperlinks. Verify that comments are sent by adding rules to your Rule Board's **Comment** column that can match code to generate a finding.
+#### Configuring PR comments in Bitbucket
```

Comment:
Just wanted to leave a note in terms of where this section is placed in the docs, it seems like Github pull request comments and Gitlab merge request comments instructions are elsewhere under the Alerts and Notifications section (https://semgrep.dev/docs/semgrep-app/notifications/#gitlab-merge-request-comments). Not sure if we want to group all the review comment instructions together in one placce.

Reply:
Yeah, I thought I should move this out elsewhere. Will do it!